gtfe/__init__.py

```diff
@@ -9,15 +9,15 @@
 RFC2822_FORMAT = '%a, %d %b %Y %T %z'
 
 
-def get_epoch_for_datetime(datetime):
+def get_Epoch_for_datetime(datetime):
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename function "get_Epoch_for_datetime" to match the regular expression ^[a-z_][a-z0-9_]{2,30}$. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=python%3AS1542)